File: PooCC01CNetz/Semana03/Python/Tarea_POO_01/Ejercicio01.py
class Ejercicio01:
    def main():
        p = int
        q = int
        p = int(input("Dame el valor (entero) de p: "))
        q = int(input("Dame el valor (entero) de q: "))
        if q == 0:
            print("error, division por cero")
        else:
            c = 0
            r = p
            while r >= q:
                r = r - q
                c = c - 1
            print(f"Valor de c: {c}, Valor de r: {r}")
    
    main()

File: PooCC01CNetz/Semana03/Java/Tarea_POO_01/Ejercicio01.java
import java.util.Scanner;
public class Ejercicio01 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int p, q;

        System.out.print("Dame el valor (entero) de p: ");
        p = scanner.nextInt();
        System.out.print("Dame el valor (entero) de q: ");
        q = scanner.nextInt();

        if (q == 0) {
            System.out.println("error, division por cero");
        } else {
            int c = 0;
            int r = p;
            while (r >= q) {
                r = r - q;
                c = c - 1;
            }
            System.out.println("Valor de c: " + c + ", Valor de r: " + r);
        }
        
        scanner.close();
    }
}